go.mod

```diff
@@ -6,9 +6,9 @@ require (
 	github.com/Masterminds/sprig/v3 v3.1.0
 	github.com/google/go-github/v31 v31.0.0
 	github.com/gorilla/mux v1.8.0
-	github.com/mattermost/mattermost-plugin-api v0.0.12-0.20200917103517-788aef584648
-	github.com/mattermost/mattermost-server/v5 v5.28.0
+	github.com/mattermost/mattermost-plugin-api v0.0.16
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20210721100405-6688195dce2a
```

Comment:
Note to reviewers: I want to bump it to version v5.38.0 when it is out as earlier versions do not contain `ChimeraOAuthProxyURL` config.

Reply:
Should the `min_server_version` also get updated?

Reply:
Older server versions should work fine it is just the new option will not work for them. I am not sure what are exact implications of bumping `min_server_version` (I do not want people running self-managed instances to force the upgrade for the plugin to work) so I would follow your lead on that.

Reply:
After double-checking, it seems that we do need to upgrade the `min_server_version` or otherwise plugin configuration fails to load. I will do it together with `go.mod` update after server release.

Reply:
Did you meant to update the `min_server_version`?

Reply:
Yes, sorry for confusion, I did not realise I phrase it pretty badly 😄